### ISSUE TITLE
Bony island of the tympanic membrane

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#11658]
@evanc1648

adjusting definition/label

Adding new term: 
- ID: HP:6001492
- label: Aberrant tympanic bony island
- parents: [Abnormal tympanic membrane morphology (HP:0040090)](https://hpo.jax.org/app/browse/term/HP:0040090)
- pmid: PMID:27428025
- def: Ectopic or heterotopic mature osseous tissue localized within the fibrous layers or structural boundaries of the tympanic membrane, independent of normal middle ear ossicular anatomy.
- comment: This feature may be observed in Fanconi Anemia and may be associated with hearing impairment.
- orcid: orcid:0000-0002-0736-9199
- github ID: 
- synonyms: Bony island of the tympanic membrane(NONE)

**Comment by evanc1648** [comment]
@pnrobinson Thank you for refining the term label and the definition. It all looks good to me. Thank you!

**Comment by pnrobinson** [comment]
thanks, closing.
